Sources/AWSLambdaRuntimeCore/LambdaRunner.swift

```diff
@@ -74,6 +74,7 @@ internal final class LambdaRunner {
                 allocator: self.allocator,
                 invocation: invocation
             )
+            logger.trace("Request", metadata: ["bytes": .string(String(buffer: bytes))])
             logger.debug("sending invocation to lambda handler")
```

Comment:
I think we should combine those two. debug is the right level imo.
What happens with payloads that aren't string representable. IIRC I can send any payload to lambda.

Reply:
Thank you @fabianfett for the feedback.

> I think we should combine those two. debug is the right level imo.
I like the idea of separating debug output from tracing. Trace adds more info, potentially long data trace, which is the case here.
But maybe it's a matter of personal preference.
I can see user preferences for having just the debug logs, while some others want to have the trace level for a limited period of time.
Given the potentially sensitive nature of the data contained in the payload (such as JWT tokens or PII), I think it is a good idea to decouple the debug and trace levels in this case.

> What happens with payloads that aren't string representable

According to the AWS Lambda service API documentation, the `payload` is always a string 
https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html

Furthermore, the max payload size for synchronous invocations is 6Mb. They're wont be huge payloads passed as input to the swift AWS Lambda Runtime.
https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html

Reply:
we can also do something as follow if we want to eliminate the two logging statements  

```
if logger.logLevel <= .trace {
    logger.trace("...")
  } else {
    logger.debug("...")
  }
}
```

Reply:
perhaps we want to trim it?